python-package/lightgbm/sklearn.py

```diff
@@ -558,18 +526,12 @@ def fit(self, X, y,
                                         callbacks=callbacks)
         return self
 
-    def predict(self, data, raw_score=False, num_iteration=0):
-        class_probs = self._Booster.predict(data,
-                                            raw_score=raw_score,
-                                            num_iteration=num_iteration)
-        if len(class_probs.shape) > 1:
-            column_indexes = np.argmax(class_probs, axis=1)
-        else:
-            column_indexes = np.repeat(0, class_probs.shape[0])
-            column_indexes[class_probs > 0.5] = 1
-        return self._le.inverse_transform(column_indexes)
+    def predict(self, X, raw_score=False, num_iteration=0):
+        class_probs = self.predict_proba(X, raw_score, num_iteration)
+        class_index = np.argmax(class_probs, axis=1)
```

Comment:
I think return of first max is OK. 
And lightgbm also use first max as predict class for multi-classification metrics.